### ISSUE TITLE
[core][200 USD] 100% coverage for core-elasticsearch

### ISSUE THREAD

**Issue by faustbrian** [ArkEcosystem/tier-0-program#3]
**About This Project**

> The difficulty level of this bounty is estimated as **intermediate**. The reward for completion of this project is set at **$200** (paid in ARK Ѧ).

_Requirements for completion:_
- Increase code coverage to 100% for the `@arkecosystem/core-elasticsearch` package which may or may not include writing Functional, Integration and/or Unit tests

_Necessary Skills:_
- [TypeScript](https://www.typescriptlang.org/)
- [Jest](https://jestjs.io/)
- [Elasticsearch](https://www.elastic.co/)

**Applying For This Project**

To apply for this project, please read more about the process below before leaving a comment with the following:

1. A brief proposal relating to the project (e.g. Your experience, how you wish to approach the project, estimated time to completion etc.)
2. Your name and a contact email address (or another form of contact, eg. Slack username) 

**The Process**

Applicant’s will only be permitted to participate in one Tier 0 project at any one time. You can apply for as many projects as you like, but will not be eligible to be accepted until your open projects have been completed. In the meantime the projects may be awarded to other available applicants.

If your application for the project is successful we’ll contact you using the details you provided, assign the project to you and mark it as in progress. You’ll be required to check in regularly with the ARK team to discuss progress and development. We’ll also use these updates to ensure that the PR is being completed to a satisfactory standard. If we determine that the project isn’t being completed to a satisfactory standard and that this cannot be achieved with minor amends or within a satisfactory timeframe, we hold the right to dismiss the PR and reopen the project to other applicants.

To deliver the project your PR must be submitted to the respective repo. ARK repo maintainers will then leave comments and request amends if necessary. All amends requested by maintainers must be resolved before completion, and all projects must comply with our code standards and contribution guidelines (found at: https://docs.ark.io/guidebook/contribution-guidelines/). After you’ve submitted your project, we’ll review against the brief to ensure it meets the criteria and deliverables.

Once your project is approved as complete a member of the ARK team will follow up to discuss payment. You’ll need to submit an invoice and payment will be issued in (Ѧ) ARK (equivalent to the USD reward value assigned to the project), sent to an address of your choosing. We may need to request additional information regarding your identity in order to comply with our own internal Know Your Customer (KYC) guidelines.

_If you’re unsuccessful in applying for this particular project, please consider applying for others. Thanks for your interest and good luck with your application!_

**Comment by faustbrian** [comment]
This will be on hold until the release of Core 3.0 as a lot of reworks are happening.

**Comment by faustbrian** [comment]
Closing as it will be done during the development of Core 3.0.
